AttitudeManager/FlightModes/Inc/flightmode.hpp

```diff
@@ -10,15 +10,17 @@
 #define ZPSW3_AM_CONTROL_INTERFACE_HPP
 
 #include "CommonDataTypes.hpp"
-
+#include <PID.hpp>
 namespace AM {
 
 class Flightmode {
    public:
     virtual ~Flightmode() = default;
 
-    virtual AttitudeManagerInput run(const AttitudeManagerInput& input) = 0;
+    virtual AttitudeManagerInput run(const AttitudeManagerInput& input, AxisPIDs) = 0;
```

Comment:
we shouldn't be passing AxisPIDs in as a function to our AM ::run function....

We should be able to grab the values from private class variables.

Those PID's should only be set using the updatePIDGains or in the initialization function.